src/helpers/prometheus/index.ts

```diff
@@ -0,0 +1,21 @@
+import client from 'prom-client';
+import { queryResponseTimeMetric, totalApiCallsMetric, failedApiCallsMetric, successApiCallsMetric } from './metrics'
+import { NextFunction } from 'express';
+
+const incrementApiCalls = (labels: Record<string, any> = {}) => totalApiCallsMetric.labels(labels).inc();
+const setQueryResponseTime = (duration: any, labels: Record<string, any> = {}) => queryResponseTimeMetric.labels(labels).set(duration);
+const incrementFailedApiCalls = (labels: Record<string, any>) => failedApiCallsMetric.labels(labels).inc();
+const incrementSuccessfulApiCalls = (labels: Record<string, any>) => successApiCallsMetric.labels(labels).inc();
+
+const metricsScrapeHandler = async (req: any, res: any, next: NextFunction) => {
+    try {
+        res.set('Content-Type', client.register.contentType);
+        const metrics = await client.register.metrics()
```

Comment:
Metrics Registration should happen only once not every time the scrapper API is called.

Refer to web console implementation